README.md

```diff
@@ -1,2 +1,25 @@
 # QuickFix
-bref décrire l'appli ici
+
+**QuickFix** is a platform designed to connect users who need tasks done with skilled manual workers. Whether it's painting, plumbing, or any other service, QuickFix makes it easy to find the right person for the job.
+
+## Features
+- **User-friendly Interface**: Easy-to-navigate platform for users and workers alike.
+- **Worker Profiles**: Manual workers can create detailed profiles showcasing their experience, services, and rates.
+- **Rating & Reviews**: Users can leave feedback for workers after a job is completed, ensuring quality and trust in the system.
+
+## Supported Services
+Currently, the app supports connecting users with:
+- Painters
+- Plumbers
+- Gardeners
+- Electricians
+- Handymen
+- And more!
+
+## Development Progress
+The app is still under active development. For more detailed information about the development process, current progress, and updates, please refer to our [Wiki](https://github.com/arygm/QuickFix/wiki).
+
```

Comment:
This is good. But maybe we could add a sub-section describing what is currently being done, which could indicate the lecturer the future features of our app.

Reply:
The [Wiki](https://github.com/arygm/QuickFix/wiki) is more appropriate to what is currently being done but I will update the `README.md` as need be when new features are implemented.